qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/PersonFormPage.java

```diff
@@ -28,7 +28,6 @@ public PersonFormPage(Page parent) {
 
 	public void savePerson() {
 		clickOn(SAVE_PERSON);
-		waitForTextToBePresentInElement(OPENMRS_MSG, "Person saved");
```

Comment:
Any reason why removing this

Reply:
Oh removed it while testing something and forgot to put it back. Thanks